src/powershell/install-category.ps1

```
@@ -0,0 +1,15 @@
+$ErrorActionPreference = "SilentlyContinue"
+
+$category = $env:CATEGORY
+Write-Output "Installing packages for category $category"
+
+[xml]$xml = Get-Content "C:\\packages\\$category.config"
+
+$xml.SelectNodes('//packages/package') | ForEach-Object {
+    $packageID = $_.id
+    Write-Output "### current package: $packageID ####################################################"
+    try { choco upgrade $packageID --yes --no-progress }
+    catch { }
```

Does this just silently catch any errors with `choco` and continue on? I believe any failures during installation should fail the entire build process to correctly set expectations for what a successful build means.

---

Yes, this code did silently ignore all errors. This was useful while debugging issues with packer build timeouts. I agree that builds should fail if there is a problem installing choco packages. Thank you for this improvement.

Resolved here: https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/ef0a55e123aceea4f314f0696eb7371d40b8b9d8